style.css

```diff
@@ -0,0 +1,616 @@
+}
+
+.page-par {
+  color: #4e4e50;
+  border: 4px solid #ffff;
+  background-color: #f1ecec;
+  width: 80%;
+  padding: 10px;
+  margin-bottom: 60px;
+}
+
+.page-par-about {
+  color: #272a31;
+  width: 90%;
+  text-align: center;
+  padding: 20px;
+  margin: 30px auto;
+  background-color: white;
+  border: 1px solid #d3d3d3;
+}
+
+.contact-us {
+  color: #272a31;
+  width: 70%;
+  text-align: center;
+  margin: 0 auto;
+  margin-bottom: 10px;
+}
+
+.email-link {
+  color: #272a31;
+  text-align: center;
+}
+
+.section-2 {
+  color: white;
+  text-align: center;
+}
+
+.about-h2 {
+  text-align: center;
+}
+
+.about-par {
+  text-align: center;
+  width: 59%;
+  margin: 0 auto;
+  padding-top: 30px;
+}
+
+.about-logo {
+  height: 300px;
+  border: 1px solid #d3d3d3;
+  margin: 20px auto;
+  padding: 80px 150px;
+}
+
+.past-projects {
+  margin: 0 20px;
+  margin-top: 100px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.section-3 {
+  margin-bottom: 40px;
+}
+
+.project-1 {
+  background-image: url("img/proj-1.png");
+  width: 30%;
+  height: 150px;
+  margin: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.project-2 {
+  background-image: url("img/proj-2.jpg");
+  width: 30%;
+  height: 150px;
+  margin: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.bold {
+  color: #272a31;
+  font-weight: bolder;
+  font-size: 35px;
+}
+
+.bottom-text {
+  color: #d3d3d3;
+}
+
+.main-section {
+  background-image: url("img/grey\ img.png");
+  text-align: center;
+  padding-top: 20px;
+  background-size: cover;
+  padding-bottom: 50px;
+}
+
+.main-section-h1 {
+  text-align: center;
+  margin-bottom: 10px;
+  color: white;
+}
+
+.orange-underline {
+  width: 70px;
+  margin: 0 auto;
+  border-bottom: 5px solid #ec5242;
+  margin-bottom: 20px;
+}
+
+.blocks {
+  display: flex;
+  list-style: none;
+  margin: 10px 10%;
+  margin-bottom: 50px;
+}
+
+.blocks li {
+  margin: 10px;
+  background-color: #3d3a3a7c;
+  padding: 10px 0;
+}
+
+.blocks h2 {
+  color: #ec5242;
+}
+
+.blocks p {
+  color: #d3d3d3;
+}
+
+.see-more {
+  color: #d3d3d3;
+}
+
+.fas {
+  color: #d3d3d3;
+  padding: 10px 10px;
+}
+
+.speaker-header {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.speaker-list {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-top: 60px;
+  padding: 10px 10%;
+}
+
+.speaker {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  width: 45%;
+  margin: 15px;
+}
+
+.speaker img {
+  height: auto;
+  width: 150px;
+}
+
+.grey-underline {
+  width: 30px;
+  margin-top: 15px;
+  border-bottom: 2px solid grey;
+}
+
+.job-position {
+  color: #ec5242;
+  font-style: italic;
+}
+
+.partner {
+  text-align: center;
+  background-color: #534f4e;
+  padding: 40px 10px;
+}
+
+.partner-header {
+  margin-bottom: 5px;
+  color: white;
+}
+
+.bottom-logos {
+  list-style: none;
+  display: flex;
```

Comment:
* selector class can be used here to avoid repetition of code because It bloats your codebase if you have duplicate code all over the place

Reply:
- [ ] You can check [this article](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) to learn about DRY principles to make your code more clean and maintainable.

Reply:
Sorted